**Task : 2**

**Data Splitting**

In [1]:
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split

# Load the Iris dataset
iris = load_iris()
X = iris.data
y = iris.target

# Split the dataset into training and testing sets using an 80-20 split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Print the number of samples in the training and testing sets
print(f"Number of samples in the training set: {X_train.shape[0]}")
print(f"Number of samples in the testing set: {X_test.shape[0]}")


Number of samples in the training set: 120
Number of samples in the testing set: 30


**Explanation of Output:**

In the output:

- **Number of samples in the training set: 120**: This means that 120 data points from the Iris dataset were assigned to the training set. The training set is used to train the machine learning model.

- **Number of samples in the testing set: 30**: This indicates that 30 data points were allocated to the testing set. The testing set is used to evaluate the performance of the trained model.

The total number of samples in the Iris dataset is 150. The 80-20 split divides these 150 samples into 120 for training (80% of 150) and 30 for testing (20% of 150).